.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -10,6 +10,10 @@ Reviewing guidelines: https://nixos.org/manual/nixpkgs/unstable/#chap-reviewing-
 
 ###### Motivation for this change
```

Comment:
Should the heading still be "Motivation"? 
> mo·ti·va·tion
> noun: motivation; plural noun: motivations
>    the reason or reasons one has for acting or behaving in a particular way.

I can't really think of anything better but currently it is mismatched with the added instructions.

Reply:
Well, I'd argue that the upstream changes can be a driving factor, so requesting a changelog here sound very reasonable to me.

Reply:
> Should the heading still be "Motivation"?

I had the same thought about this. Would you like "Description" better?

Reply:
I like it, seems a bit more unambiguously generic and doesn't stretch the word.
```suggestion
###### Description of this change
```

Reply:
Probably worth considering if that still reflects the original motivation behind that section. 0bd54a6d5d4eba361c5b8ff04dd3c02d6dbfb259

Reply:
I do think so. The goal was to have people write at least *something* about their PR, instead of just going through the check list.

Reply:
Done, changed to "description".

Reply:
The original author wrote the commit message:

> I'm seeing a lot of PRs that only include the checklist, when the main thing I care about when I'm reviewing a PR is what the author was thinking when they decided to make the change.

ie thoughts before writing the change, whereas a description is summarizing after the change. That is a significant difference.

> currently it is mismatched with the added instructions.

Isn't this a feature, as we want people to think about what they can write?